bootcd/rpms/genesis_scripts/src/run-genesis-bootloader

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+GENESIS_MODE="$(grep -o -e 'GENESIS_MODE=[^ ]*' /proc/cmdline | cut -c14-)"
+
+# setup the ruby environment we need
+source /etc/profile.d/rvm.sh
+rvm list
+ruby --version
+
+if [[ ! -z $GENESIS_MODE ]] && [[ ! -e "/var/run/bootloader-autorun.lock" ]]
+  then
+    touch /var/run/bootloader-autorun.lock
+    # disable task execution prompting when doing initial target run
+    echo "Running Genesis"
+    GENESIS_PROMPT_TIMEOUT=0  /usr/bin/genesis-bootloader 2>&1 | tee /var/log/genesis-bootloader.log
+fi
+echo "Genesis done. Dropping to a shell"
+exec $SHELL
```

Comment:
Still needs some work. The behavior of this line isn't what I would have expected.